lib/traject/readers/folio_postgres_reader.rb

```diff
@@ -255,37 +261,40 @@ def sql_query(conditions, addl_from: nil)
             )
       FROM sul_mod_inventory_storage.instance vi
       LEFT JOIN sul_mod_inventory_storage.holdings_record hr
-         ON hr.instanceid = vi.id
+          ON hr.instanceid = vi.id
       LEFT JOIN sul_mod_inventory_storage.item item
-         ON item.holdingsrecordid = hr.id
+          ON item.holdingsrecordid = hr.id
+      -- Instance's statistical code
+      LEFT JOIN LATERAL jsonb_array_elements_text(vi.jsonb -> 'statisticalCodeIds') uuid_stats_code(id) ON TRUE
```

Comment:
I created https://github.com/sul-dlss/searchworks_traject_indexer/pull/987 so that we have some reproducable way to evaluate the performance of query changes.

Reply:
This indeed causes the test `./spec/lib/traject/readers/folio_postgres_reader_spec.rb` to fail.  I will think about another approach.

Reply:
Hm. I was thinking the performance was in the margin of error (although I'm consistently getting timeouts everywhere now).. I was seeing ~45s for `main` and ~60s for this branch, but smaller time-slices were pretty much identical.

Reply:
@cbeer I added a second commit that has the same effect, but relies on a second query rather than adding joins.